Dockerfile

```diff
@@ -0,0 +1,21 @@
+
+FROM node:18-alpine
+
+
+RUN npm install -g nodemon
```

Comment:
if we add nodemon as dependency in package.json, then we don't need to install it globally here